From 68382242986b24dbf4329586591c102b3383ff95 Mon Sep 17 00:00:00 2001
From: Jon Eyolfson <jon@eyolfson.com>
Date: Wed, 31 Mar 2010 00:53:48 -0400
Subject: [PATCH] Removed main menu arrow

---
 gnome-panel/panel-menu-button.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gnome-panel/panel-menu-button.c b/gnome-panel/panel-menu-button.c
index e9676b3..905c24d 100644
--- a/gnome-panel/panel-menu-button.c
+++ b/gnome-panel/panel-menu-button.c
@@ -648,7 +648,7 @@ panel_menu_button_load (const char  *menu_path,
 			       "tooltip", tooltip,
 			       "use-menu-path", use_menu_path,
 			       "use-custom-icon", use_custom_icon,
-			       "has-arrow", TRUE,
+			       "has-arrow", FALSE,
 			       NULL);
 
 	info = panel_applet_register (GTK_WIDGET (button), NULL, NULL,
-- 
1.7.0.3

